vcpkg/users/triplets.md

```diff
@@ -210,6 +210,20 @@ tracked as part of the package abis.
 This will cause [Binary Caching](binarycaching.md) to reuse builds from older or
 newer compilers.
 
+### VCPKG_USE_HOST_TOOLS
```

Comment:
I'm not sure why this is in the 'triplets' section given that I don't believe it's interpreted there.

The only mentions I see of this variable in the product are in `scripts/buildsystems/vcpkg.cmake` and triplet variables will not be defined there.

Moreover, this controls nothing about vcpkg; it controls which 'tools' directories are added to CMAKE_PROGRAM_PATH in the *user's* build system.

(Of course if someone wants to set this in their triplet they can do it through VCPKG_CMAKE_CONFIGURE_OPTIONS)